examples/deployment/flyio-example/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11-bullseye
```

Comment:
Maybe consider bookworm! I am deploying to fly, too using `python:3.11-slim-bookworm`. Works nicely

Reply:
Oh, that's really nice. The images are quite large, so this should help.